src/views/components/label/index.njk

```diff
@@ -0,0 +1,55 @@
+---
+title: Label
+description: Labels are paired with inputs, textareas, radios, checkboxes, and selects to describe a form input.
+layout: default
+eleventyNavigation:
+  key: Label
+  section: components
+---
+
+{% from "example.njk" import example %}
+{{ example({group: "components", item: "label", example: "default", html: true, open: false, size: "s" }) }}
+
+<section class="my-8">
+    <h2 class="mb-4">When to use this component</h2>
+    <p>Labels are paired with inputs, textareas, radios, checkboxes, and selects to describe a form input.</p>
+</section>
+
+<section class="mb-8">
+    <h2 class="mb-4">When not to use this component</h2>
+    <p>Open-ended questions can be difficult for users to answer, so consider breaking down the question into simpler questions that can be answered using a single 
+    line input or by selecting an answer using, for example, radios.</p>
+</section>
+
+<section class="mb-8">
+    <h2 class="mb-4">How to use this component</h2>
+    <p>Generally speaking you <strong>should not</strong> need to implement a label yourself as the form inputs have them imported in their respective templates.</p>
```

Comment:
I think with Hamish's comment this will now be removed as the actual content is eQ-specific, but yes the comma would've been grammatically correct :)